setup.cfg

```diff
@@ -12,7 +12,7 @@ install_requires =
     msgpack
     nanomsg
     pyzmq
-    selenium
+    selenium==3.141.0
```

Comment:
Rather than pinning this to a specific version, it might be better to use a range like `selenium<4`.  I am assuming that the new (last week) version 4 is what breaks things.  By pinning to a range, we ensure that we'd continue to pull in updates to the 3.X release train in case of future bugfixes or security patches.  (I don't know how likely there are to be any such changes in the future -- possibly unlikely.  But we don't need to rule them out from the get-go.)  WDYT?

I'd also be interested to hear what the problems have been in moving to a newer selenium-python version, and getting the changes needed to migrate onto the mite roadmap (such as it is) -- but that's very much a "nice to have" and not a requirement to merge the PR.

Reply:
I agree entirely that pinning a version is probably not the best idea! However, it looks like anything above 4.0.0a5 drags in a whole bunch of dependencies (which includes the need to install the rust compiler, which in itself seems to create issues), so, on balance, I suspect, that for the near future at least, it is probably better to  have a working mite build which has a pinned selenium version, rather than have a broken build. I also think that further investigation as to what benefits the later selenium versions have would be worthwhile.
The version I have pinned to, is the last known good "release" version. Whereas, all the 4x versions, up to HEAD, are pre-release / alpha, so, I am a bit reticent to use those versions in prod. Counter thoughts?